README.md

```diff
@@ -264,15 +267,68 @@ For a [picker](#built-in-pickers) specific remapping, it can be done by setting
 its `attach_mappings` key to a function, like this
 
 ```lua 
+local actions = require('telescope.actions')
 -- Picker specific remapping
 ------------------------------
-require('telescope.builtin').fd({
-  attach_mappings = function(prompt_bufnr, map)
-    map('i', '<esc>', actions.close)
-    map('i', '<c-i>', my_cool_custom_action)
-  end
+require('telescope.builtin').fd({ -- or new custom picker's attach_mappings field:
+  attach_mappings = function(prompt_bufnr)
+    -- This will replace goto_file_selection_edit no mather on which key it is mapped by default
+    actions.goto_file_selection_edit:replace(function()
+        -- Code here
```

Comment:
Can you add here:   
```lua
local entry = actions.get_selected_entry()
actions.close(prompt_bufnr)
```
so that people know they can still get the selected_entry and close the action here. Does this make sense to add?

Reply:
You mean under attach_mappings? perhaps you can add comment saying that entry gotten from action.get_selected_entry?. Also, where do `entry` get used?

Reply:
So basically i would change it to
```lua
actions.goto_file_selection_edit:replace(function()
  local entry = actions.get_selected_entry()  
  actions.close(prompt_bufnr)
  print(vim.inspect(entry))
  -- Code here
end)
```

That basically show how to get the selected entry table and users can than do all their fancy vim/lua stuff they wanna do.

Reply:
Oh that so much better. Thanks @Conni2461